frontend/js/src/common/listens/MBIDMappingModal.tsx

```diff
@@ -285,21 +262,85 @@ export default NiceModal.create(({ listenToMap }: MBIDMappingModalProps) => {
                 </div>
               )}
             </div>
-            <div className="modal-footer">
-              <button
-                type="button"
-                className="btn btn-default"
-                onClick={closeModal}
+            <div className="modal-footer" style={{ textAlign: "left" }}>
+              <div
+                className="mb-10"
+                style={{
+                  display: "flex",
+                  justifyContent: "space-between",
+                }}
               >
-                Cancel
-              </button>
-              <button
-                type="submit"
-                className="btn btn-success"
-                disabled={!selectedRecording}
-              >
-                Add mapping
-              </button>
+                <div>
+                  <Link
+                    type="button"
+                    className="btn btn-info"
+                    to="/settings/link-listens"
+                    onClick={closeModal}
+                  >
+                    <span
+                      className="fa-layers fa-fw"
+                      style={{ marginRight: "0.5em" }}
+                    >
+                      <FontAwesomeIcon icon={faLink} />
+                      <FontAwesomeIcon
+                        icon={faLink}
+                        color="#FFFFFFA1"
+                        transform="right-5"
+                      />
+                      <FontAwesomeIcon
+                        icon={faLink}
+                        color="#ffffff42"
+                        transform="right-10"
+                      />
+                    </span>
+                    &nbsp; Mass-link listens tool
+                  </Link>
+                </div>
+                <div style={{ textAlign: "right" }}>
+                  <button
+                    type="button"
+                    className="btn btn-default"
+                    onClick={closeModal}
+                  >
+                    Cancel
+                  </button>
+                  <button
+                    type="submit"
+                    className="btn btn-success"
+                    disabled={!selectedRecording}
+                  >
+                    Add mapping
+                  </button>
+                </div>
+              </div>
+              <div className="small help-block">
```

Comment:
All three of these lines need the help-block class removed, for placement:
```suggestion
              <div className="small">
```